### PR TITLE
HTAN data model visualization website instance

### PR DESCRIPTION
**Pull request by milen-sage**
Request an instance optimized to run a web site with an elastic search backend:
two open ports 8000 and 9200.

### CODE REVIEW

**Comment by milen-sage** [comment]
> Use new template.

I think I used the correct one now?

File: config/prod/htan-data-model-web-ec2.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
+stack_name: htan-data-model-web-ec2
+sceptre_user_data:
+  Distro: "aws"
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  OpenPorts: ["8000", "9200"]
```
Comment (zaro0508) [inline config/prod/htan-data-model-web-ec2.yaml]: this is ok.  was wondering if this website should only be accessible by sage employees or should it be accessible by people outside of sage as well?

File: config/prod/htan-data-model-web-ec2.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.medium"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "24"
+  # (Optional) Base instance on AMI (default: ami-082278746f893d99c) (supported distros: AWS linux)
+  # AMIId: "ami-0de53d8956e8dcf80"
+  # (Optional) Run EC2 in a subnet (default: PrivateSubnet) (valid values: PrivateSubnet or PublicSubnet)
+  VpcSubnet: "PublicSubnet"
```
Comment (zaro0508) [inline config/prod/htan-data-model-web-ec2.yaml]: if this is only for sage employees then we prefer "PrivateSubnet"

File: config/prod/htan-data-model-web-ec2.yaml (hunk 1 above)
Comment (milen-sage) [reply config/prod/htan-data-model-web-ec2.yaml]: The site should be accessible outside Sage as well.
It'd be great to assign a domain name as well eventually, we'll have to decide on the name first w/in HTAN.

**Pull request merged**